app/controllers/demos_controller.rb

```diff
@@ -29,4 +31,14 @@ def campaign
   def demo_params
     params.permit(:template, :theme, :campaign_id)
   end
+
+  def load_campaigns
+    @campaigns = if authorized_user&.can_admin_system?
+      Campaign.all.order(name: :asc)
+    elsif current_user&.campaigns
+      current_user.campaigns.order(name: :asc)
+    else
+      []
```

Comment:
One of the original goals of this demo page was so that everyone - publishers, potential advertisers, etc. - would be able to see our ads in action with all of the available templates. That’s why it was scoped to only be querying a specific fallback campaign we setup specific for this. 

I just want to make sure that you’re intending to remove this feature for those users. If that’s the case, we should put some authorizations in place so only current advertisers can see this page.

Reply:
I agree. This still works great if you are not authenticated. If you *are* authenticated, it will allow you to select the campaigns from your list of campaigns (if available) and if not, it will fall back to the default.

Reply:
Ahhh ok I read this wrong at first bc I’m on mobile and GitHub folded part of the code which made arrive at that incorrect understanding of this change. Disregard :) 

However do you need a break on 37? It’s possible for an admin to meet the if and elsif condition (admin who has also created fallback campaigns). Could be wrong but something about this is definitely making me suspicious